### PR TITLE
add rhacm21 issuer secret workaround script

### PR DESCRIPTION
**Pull request by huang-cn**
for RHACM 2.1, it needs to copy the issuer secret to the open-cluster-management-issuer namespace as a workaround.

### CODE REVIEW

File: scripts/cp4mcm-rhacm21-cp-issuer-secret.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+r=`kubectl -n $ISSUER_NS get secret ${ISSUER_SECRET} 2>/dev/null | grep $ISSUER_SECRET`
+if [ ! -z "$r" ]; then
+  echo "secret already exist."
+  exit 0
+fi
+
+while [[ -z "$result" && !( $TESTCOUNT == 0 ) ]]
+do
+  sleep $INTERVAL
+  result=`kubectl get secret ${ISSUER_SECRET} -n ibm-common-services 2>/dev/null |grep -v NAME`
```
Comment (gyliu513) [inline scripts/cp4mcm-rhacm21-cp-issuer-secret.sh:17]: take `ibm-common-services` as an env var?

File: scripts/cp4mcm-rhacm21-cp-issuer-secret.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ISSUER_NS=open-cluster-management-issuer
+ISSUER_SECRET=cs-ca-certificate-secret
+INTERVAL=30
```
Comment (gyliu513) [inline scripts/cp4mcm-rhacm21-cp-issuer-secret.sh:5]: Prefer interval as 5 seconds

Comment (huang-cn) [reply scripts/cp4mcm-rhacm21-cp-issuer-secret.sh]: I've updated the interval to 5s and used variable for `ibm-common-services` namespace.

**Comment by gyliu513** [comment]
@stacyped can you help review and have this merged?

**Review by gyliu513 — APPROVED** [review]

**Comment by stacyped** [comment]
Can you update the CHANGELOG.md as well? Thank you

**Comment by huang-cn** [comment]
@stacyped the CHANGLOG.md update, could you help to review and merge?

**Pull request merged**